pkg/source/source.go

```diff
@@ -48,6 +48,8 @@ const (
 	// by the previous agent instance. Can be overwritten by all other
 	// sources (except for unspec).
 	Restored Source = "restored"
+
+	Directory Source = "Directory"
```

Comment:
Please add some Go doc string here. Also:

```suggestion
	Directory Source = "directory"
```